server/src/actors/project.rs

```diff
@@ -925,7 +925,10 @@ impl ProjectCache {
                         if !slf.shutdown.requested() {
                             // Put the channels back into the queue, in addition to channels that
                             // have been pushed in the meanwhile. We will retry again shortly.
-                            slf.state_channels.extend(channels);
+                            slf.state_channels.extend(batch);
+                            metric!(
+                                histogram("project_cache.size") = slf.state_channels.len() as u64
```

Comment:
this is duplicated above

Reply:
The value changes again, though, so the histogram needs updating.